data/hai/hai reveal 4 middle.txt

```diff
@@ -97,37 +96,37 @@ mission "Hai Reveal [A02] Symbol Identification"
 					goto cadmium
 				`	"How long have the kidnappings been happening?"`
 					goto kidnappings
-				`	"How can I help you?"`
+				`	"That's enough information. How can I help you?"`
 					goto help
 			label uninhabited
-			`	Teeneep looks at you as if you were daft. "The kidnappings happen on Hai and human worlds. The kidnappers have no reason to search an uninhabited planet for potential victims. This gives us a safe place to hide any critical people or information. We have reason to believe that if there is a coordinated organization behind this as we suspect, they would have strong reason to track us down and silence some those who have previously escaped, and probably shut us down too. We have plenty of space here to do our work and investigate what we need to without the precautions that would be necessary somewhere more populated."`
+			`	Teeneep looks at you as if you were daft. "Kidnappers have no reason to search an uninhabited planet for potential victims. This gives us a safe place to hide critical people or information. If the kidnappings are a coordinated effort as we suspect, they would track down threats to the operation, like escaped victims, or our organization. We have plenty of space here to do our work without precautions that would be necessary on a populated planet."`
 				goto questions
 			label cadmium
-			`	A Hai in a medical uniform explains, "The victims each seem to have been exposed to airborne particulates containing cadmium which must be circulating through their ships ventilation. I should also clarify that it's not strictly 'poisoning'. The level of exposure is quite low, but enough that testing shortly after being rescued does show clearly elevated levels in many who were taken.`
-			`	"No known technology of human or Hai origin exhausts anything which could create this effect. There are some microbots used in Hai medicine that are used to seal off ruptures causing internal bleeding. Their dispenser canisters have a residue of residual cadmium dust in them, but those are the only thing we even bring onto a ship that uses it and are self-contained for disposal. The molecular bonding doesn't match anyway. There are some materials involved with properties unlike anything either of us make, which suggests this is coming from another race's technology."`
+			`	A Hai in a medical uniform explains, "The victims each have been exposed to complex cadmium compounds, shortly before rescue. These compounds leave a human or Hai quickly, but the level after rescue is high enough to detect.`
```

Comment:
```suggestion
			`	A Hai in a medical uniform explains. "The victims each have been exposed to airbourne cadmium particles, which must be circulating through a ships' ventilation. Although the level of exposure is quite low, after being rescued we can detect clearly elevated levels in many who were taken.`
```

I don't know, but it seemed to have changed the meaning. But I do think the science is more than the player really wants to hear.

Reply:
The science was wrong. Cadmium remains in your system for decades. Also, there aren't different kinds of cadmium; it is an element. To have different kinds, it needs to be part of a larger thing, like a compound.

Reply:
I don't see what is wrong with what I changed. And there are isotopes of cadmium, but that's not important.

Reply:
I thought I just committed this suggestion. Not sure what happened. I'll apply your suggestion again and see what github does.

Reply:
The change is in there now, but I had to update the paragraph after it for this change to make sense.